src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -374,13 +383,93 @@ impl DragonballInner {
         Ok((old_vcpus, new_vcpus))
     }
 
+    pub(crate) fn resize_memory(
+        &mut self,
+        req_mem_mb: u32,
+        default_mem_mb: u32,
+    ) -> Result<(u32, MemoryConfig)> {
+        let had_mem_mb = default_mem_mb + self.mem_hotplug_size_mb;
```

Comment:
Seems it will bring error when if mem_sb is not equal to 0 in initial size.

If initial mem_sb != 0, current memory != default_mem_mb + self.mem_hotplug_size_mb. Thus comparing the req and had_mem_mb here will cause error...

Reply:
fixed